genie-core/src/main/java/com/netflix/genie/core/properties/JobsUsersProperties.java

```diff
@@ -31,4 +31,5 @@
 public class JobsUsersProperties {
     private boolean creationEnabled;
     private boolean runAsUserEnabled;
+    private int activeJobsLimit;
```

Comment:
Assuming you're using the java language default of 0 for integer here rather than saying it explicitly